js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -193,6 +194,34 @@ export class MiniAppBridge {
     });
   }
 
+  /**
+   * Associating getMessageUniqueId function to MiniAppBridge object.
+   */
+  getMessageUniqueId() {
+    return new Promise<string>((resolve, reject) => {
+      return this.executor.exec(
+        'getMessageUniqueId',
```

Comment:
Sorry one more comment:
Should this one send `getUniqueId` as this string instead? I was thinking this would make backwards compatibility easier. Otherwise the Mini App will have to worry about whether the Host App supports the new function name or not. Also then the Host App doesn't need to make any changes and can just keep their current `getUniqueId` implementation.

What do you think?